jsonapi/fixtures_test.go

```diff
@@ -561,51 +661,45 @@ func (n CustomMetaPost) GetCustomMeta(linkURL string) map[string]Meta {
 
 type NoRelationshipPosts struct{}
 
-func (n NoRelationshipPosts) GetID() string {
-	return "someID"
+func (n NoRelationshipPosts) GetID() Identifier {
+	return Identifier{ID: "someID", LID: "", Name: "posts"}
 }
 
-func (n *NoRelationshipPosts) SetID(ID string) error {
+func (n *NoRelationshipPosts) SetID(ID Identifier) error {
 	return nil
 }
 
-func (n NoRelationshipPosts) GetName() string {
-	return "posts"
-}
-
 type ErrorRelationshipPosts struct{}
 
-func (e ErrorRelationshipPosts) GetID() string {
-	return "errorID"
+func (e ErrorRelationshipPosts) GetID() Identifier {
+	return Identifier{ID: "errorID", LID: "", Name: "posts"}
 }
 
-func (e *ErrorRelationshipPosts) SetID(ID string) error {
+func (e *ErrorRelationshipPosts) SetID(ID Identifier) error {
 	return nil
 }
 
-func (e ErrorRelationshipPosts) GetName() string {
-	return "posts"
-}
-
-func (e ErrorRelationshipPosts) SetToOneReferenceID(name, ID string) error {
+func (e ErrorRelationshipPosts) SetToOneReferenceID(name string, ID *Identifier) error {
 	return errors.New("this never works")
 }
 
-func (e ErrorRelationshipPosts) SetToManyReferenceIDs(name string, IDs []string) error {
+func (e ErrorRelationshipPosts) SetToManyReferenceIDs(name string, IDs []Identifier) error {
 	return errors.New("this also never works")
 }
 
 type Image struct {
 	ID    string      `json:"-"`
+	LID   string      `json:"-"`
 	Ports []ImagePort `json:"image-ports"`
 }
 
-func (i Image) GetID() string {
-	return i.ID
+func (i Image) GetID() Identifier {
+	return Identifier{ID: i.ID, LID: i.LID, Name: "images"}
```

Comment:
No need to define Name, since GetName was not implemented before

Reply:
removed